common/test/acceptance/pages/studio/textbook_upload.py

```diff
@@ -59,6 +60,13 @@ def click_textbook_submit_button(self):
         self.q(css='#edit_textbook_form button[type="submit"]').first.click()
         self.wait_for_element_absence(".wrapper-form", "Add/Edit form closed")
 
+    def click_view_live_link(self):
+        """
+        Click view live link
+        """
+        element = self.q(css='a.view')[0]
+        ActionChains(self.browser).move_to_element(element).click(element).perform()
```

Comment:
I just looked over this, and it looks like I didn't end up using this method. I navigate to the page by using the visit method, so I can delete this.